udata_hydra/routes/resources.py

```diff
@@ -21,6 +23,36 @@ async def get_resource(request: web.Request) -> web.Response:
     return web.json_response(ResourceSchema().dump(dict(resource)))
 
 
+async def get_resource_status(request: web.Request) -> web.Response:
+    """Endpoint to get the current status of a resource from the DB.
+    It is the same as get_resource but only returns the status of the resource, saving bandwith and processing time.
+    Respond with a 200 status code and a JSON body with the resource status
+    If resource is not found, respond with a 404 status code
+    """
+    try:
+        resource_id: str = request.match_info["resource_id"]
+    except Exception as e:
+        raise web.HTTPBadRequest(text=json.dumps({"error": str(e)}))
+
+    resource = await Resource.get(resource_id=resource_id, column_name="status")
+    if not resource:
+        raise web.HTTPNotFound()
+
+    status: Optional[str] = resource["status"]
+    status_verbose: str = Resource.STATUSES[status]
+
+    latest_check_url: str = f"{request.scheme}://{request.host}{str(request.app.router["get-latest-check"].url_for())}?resource_id={resource_id}"
```

Comment:
Woops, quotes used in f-string don't work with python3.9

Reply:
Well spotted, fixed!